insonmnia/hub/server.go

```diff
@@ -37,13 +37,11 @@ import (
 )
 
 var (
-	ErrInvalidOrderType = status.Errorf(codes.InvalidArgument, "invalid order type")
-	ErrAskNotFound      = status.Errorf(codes.NotFound, "ask not found")
-	ErrDeviceNotFound   = status.Errorf(codes.NotFound, "device not found")
-	ErrMinerNotFound    = status.Errorf(codes.NotFound, "miner not found")
-	errDealNotFound     = status.Errorf(codes.NotFound, "deal not found")
-	errTaskNotFound     = status.Errorf(codes.NotFound, "task not found")
-	errImageForbidden   = status.Errorf(codes.PermissionDenied, "specified image is forbidden to run")
+	ErrDeviceNotFound = status.Errorf(codes.NotFound, "device not found")
```

Comment:
oh come on :(

Reply:
It's not mine! It's indentation via gofmt!
(BTW I do not remove this constant because of it still used somewhere in the tests)